File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/FlashButtonFragment.java
package com.regula.facesample.items.customization.fragment;

import android.view.View;
import android.widget.ImageButton;

import androidx.annotation.NonNull;

import com.regula.facesample.R;
import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 7.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class FlashButtonFragment extends FaceDefaultUiFragment {

    View mFlashLightBtn;

    @Override
    public View getFlashLightButton(@NonNull View view) {
        mFlashLightBtn = super.getFlashLightButton(view);
        return mFlashLightBtn;
    }

    @Override
    public void updateFlashLightButton(boolean isLightOn) {
        if (mFlashLightBtn == null)
            return;

        if (mFlashLightBtn instanceof ImageButton)
            ((ImageButton) mFlashLightBtn).setImageDrawable(getResources().getDrawable(isLightOn ? R.drawable.flash_light_on : R.drawable.flash_light_off));
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/fragments/FlashButtonFragment.kt
package com.regula.facesamplekotlin.fragments

import android.view.View
import android.widget.ImageButton
import androidx.core.content.ContextCompat
import com.regula.facesamplekotlin.R
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class FlashButtonFragment : FaceDefaultUiFragment() {

    var mFlashLightBtn: View? = null

    override fun getFlashLightButton(view: View): View? {
        mFlashLightBtn = super.getFlashLightButton(view)
        return mFlashLightBtn
    }

    override fun updateFlashLightButton(isLightOn: Boolean) {
        if (mFlashLightBtn == null) return

        if (mFlashLightBtn is ImageButton) (mFlashLightBtn as ImageButton).setImageDrawable(
            context?.let { ContextCompat.getDrawable(it,
                if (isLightOn) R.drawable.flash_light_on else R.drawable.flash_light_off) }
        )
    }
}
